tempto-examples/bin/run_on_docker.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -ex
+
+# http://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-osx
+function absolutepath() {
+    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
+
+function retry() {
+  END=$(($(date +%s) + 600))
+
+  while (( $(date +%s) < $END )); do
+    set +e
+    "$@"
+    EXIT_CODE=$?
+    set -e
+
+    if [[ ${EXIT_CODE} == 0 ]]; then
+      break
+    fi
+    sleep 5
+  done
+
+  return ${EXIT_CODE}
+}
+
+function hadoop_master_container(){
+  docker ps --format '{{.Names}}' | grep hadoop-master
+}
+
+function check_hive() {
+  docker exec $(hadoop_master_container) hive -e 'show tables'
+}
+
+function docker_compose_network() {
+  docker network ls | grep docker | cut  -f 1 -d ' '
```

Comment:
I do not have such network on mac.
Does it appear only if I run some docker compose?

Also I do not like this `grep`. It is very error prone. 
Can't we use `docker network inspect ...` instead?